base_layer/core/src/transactions/transaction.rs

```diff
@@ -130,6 +145,29 @@ impl OutputFeatures {
     }
 }
 
+impl ConsensusEncoding for OutputFeatures {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, io::Error> {
+        let mut written = self.flags.consensus_encode(writer)?;
+        written += writer.write_varint(self.maturity)?;
```

Comment:
Maybe something like this:
```suggestion
        written += writer.write_varint(1)?; // version so that it can be decoded with the correct version
        written += writer.write_varint(self.maturity)?;
```

Reply:
Cool will add / hard code to 1